### PR TITLE
feat: report card info

### PR DESCRIPTION
**Pull request by hunjixin**
### Summary

This pull request makes the following changes:

- [ ] Change 1
- [ ] Change 2

Explain the motivation for making these changes. Does this pull request implement a feature or fix a bug? Is it a docs change or a typo fix?

### Task/Issue reference

Closes: add_link_here

### Test plan

Please describe how reviewers can test the changes in this pull request. If the test plan is challenging to explain in text alone, a short video demonstration may be included.

### Details (optional)

Add any additional details that will help to review this pull request.

### Related issues or PRs (optional)

Add any related issues or PRs.

### CODE REVIEW

File: pkg/resourceprovider/card.go [hunk 1]
```diff
@@ -51,13 +49,23 @@ func PostCard(id string, challange string, difficulty string) error {
...
-			Str("pow->event", "PowNewPowRound").
+		log.Error().
 			Msgf("execute temp file failed: %v", err)
 		return err
 	}
 
 	// Print the output
 	println(string(output1))
+
+	cardInfo := make(map[string]string)
```
Comment (hunjixin) [inline pkg/resourceprovider/card.go:60]: @arsen3d can i parse the output you the binary?

Comment (hunjixin) [reply pkg/resourceprovider/card.go:60]: i notice you make a change to post payload in the binary, so can i close this pr?

File: pkg/resourceprovider/card.go [hunk 2]
```diff
@@ -51,13 +49,23 @@ func PostCard(id string, challange string, difficulty string) error {
...
 	if err != nil {
-		log.Debug().
-			Str("pow->event", "PowNewPowRound").
+		log.Error().
 			Msgf("execute temp file failed: %v", err)
 		return err
 	}
 
 	// Print the output
 	println(string(output1))
```
Comment (narbs91) [inline pkg/resourceprovider/card.go:58]: Do we need to keep print the output if we are saving the card info?

File: pkg/resourceprovider/card.go (hunk 1 above)
Comment (narbs91) [reply pkg/resourceprovider/card.go:60]: I believe you have to close the channel too no?

**Comment by hunjixin** [comment]
close this pr due to report already be done in the binary,

https://github.com/Lilypad-Tech/card/blob/b3c56c94638a9ef5f2121ba0d2b12ca672781751/card.go#L219

**Pull request closed without merging**